torchtitan/checkpoint.py

```diff
@@ -473,15 +487,28 @@ def load(self, step: int = -1) -> bool:
 
         # We won't have optimizer states to load, if we are loading a seed checkpoint
         states = {"model": self.states["model"]} if step == 0 else self.states
+        # PyTorch bug: (pytorch/pytorch#138575)
```

Comment:
This issue is closed: https://github.com/pytorch/pytorch/pull/138575, is the idea that we use this for compatibility with older pytorch versions?